core/src/main/scala/kafka/cluster/Broker.scala

```diff
@@ -79,17 +80,21 @@ object Broker {
             else if (version == 1) {
               val host = brokerInfo("host").asInstanceOf[String]
               val port = brokerInfo("port").asInstanceOf[Int]
-              Map(SecurityProtocol.PLAINTEXT -> new EndPoint(host, port, SecurityProtocol.PLAINTEXT))
+              val securityProtocol = SecurityProtocol.PLAINTEXT
+              val endPoint = new EndPoint(host, port, ListenerName.forSecurityProtocol(securityProtocol), securityProtocol)
+              Seq(endPoint)
             }
             else {
+              val securityProtocolMap = brokerInfo.get("listener_security_protocol_map").map(
```

Comment:
Is there a convenient place to put a constant for this field? Also, the second `map` makes the lines below a bit tough to follow. Could we do this instead:
```scala
val securityProtocolMap = brokerInfo.get("listener.security.protocol.map").map(
  _.asInstanceOf[Map[String, String]].map{ case (listenerName, securityProtocol) =>
  new ListenerName(listenerName) -> SecurityProtocol.forName(securityProtocol)
})
```